protocol_gateways.tf

```diff
@@ -18,15 +18,15 @@ module "protocol_gateways" {
   nics_numbers          = var.protocol_gateway_nics_num
   secondary_ips_per_nic = var.protocol_gateway_secondary_ips_per_nic
   backend_lb_ip         = google_compute_forwarding_rule.google_compute_forwarding_rule.ip_address
-  install_weka_url      = var.install_url != "" ? var.install_url : "https://$TOKEN@get.weka.io/dist/v1/install/${var.weka_version}/${var.weka_version}"
+  install_weka_url      = var.install_weka_url != "" ? var.install_weka_url : "https://$TOKEN@get.weka.io/dist/v1/install/${var.weka_version}/${var.weka_version}"
   machine_type          = var.protocol_gateway_machine_type
   yum_repo_server       = var.yum_repo_server
   sa_email              = var.sa_email
   vm_username           = var.weka_username
   assign_public_ip      = var.assign_public_ip
   disk_size             = var.protocol_gateway_disk_size
   frontend_num          = var.protocol_gateway_frontend_num
-  weka_token_id         = google_secret_manager_secret.secret_token.id
+  weka_token_id         = google_secret_manager_secret.secret_token[0].id
   weka_password_id      = google_secret_manager_secret.secret_weka_password.id
```

Comment:
this one as well should have [0]

Reply:
Not sure. It doesn't rise an issue on apply and I see its usage in other resource code as it is eg here https://github.com/weka/terraform-gcp-weka/blob/5393c1cd29c9f14847bd09f6411b2e990460e7ce/secret_manager.tf#L15

Reply:
checked the code, I see:
```
resource "google_secret_manager_secret" "secret_token" {
  count     = var.get_weka_io_token != "" ? 1 : 0
  ```
  while weka password don't have a count, so you are right.
  ```
  resource "google_secret_manager_secret" "secret_weka_password" {
  secret_id = "${var.prefix}-${var.cluster_name}-password"
  ````